products/visualstudio.md

```diff
@@ -6,6 +6,7 @@ permalink: /visual-studio
 alternate_urls:
 -   /visualstudio
 releasePolicyLink: https://learn.microsoft.com/visualstudio/productinfo/vs-servicing
+changelogTemplate: https://learn.microsoft.com/lifecycle/products/visual-studio-__CODENAME__
```

Comment:
```suggestion
changelogTemplate: https://learn.microsoft.com/visualstudio/releases/__CODENAME__/release-notes-v__RELEASE_CYCLE__
```
I have found this for the release notes until 2019.
2017 has other links: https://learn.microsoft.com//visualstudio/releasenotes/vs2017-relnotes-history

2015 again ohter: https://learn.microsoft.com/visualstudio/releasenotes/vs2015-update3-vs
2013: https://learn.microsoft.com/visualstudio/releasenotes/vs2013-update5-vs
2012: https://learn.microsoft.com/visualstudio/releasenotes/vs2012-update4-vs
2010: https://learn.microsoft.com/visualstudio/releasenotes/vs2010-sp1-vs

Reply:
Done.